great_expectations_cloud/agent/agent.py

```diff
@@ -69,11 +70,19 @@ class GXAgent:
     user events triggered from the UI.
     """
 
+    _PYPI_GX_AGENT_PACKAGE_NAME = "great_expectations_cloud"
+
     def __init__(self: Self):
-        print("Initializing the GX Agent")
+        agent_version: str = self._get_current_gx_agent_version()
+        print(f"Initializing the GX Agent version: {agent_version}.")
```

Comment:
Oops, didn't realize I added a bullet point for this to the AC; my bad. I originally planned to handle this in https://greatexpectations.atlassian.net/browse/ZEL-301. In any case we are ahead of the game; thanks for doing this. 

I added a note to ZEL-301 about the formatting. I think throwing parentheses around the version info looks nice. No changes requested now; we can handle in that ticket!

Reply:
Thanks for catching this @JennyTee. Yes, looks like we can handle in ZEL-301